disnake/ui/item.py

```diff
@@ -81,7 +81,7 @@ class Item(WrappedComponent, Generic[V_co]):
     The current UI items supported are:
 
     - :class:`disnake.ui.Button`
-    - subtypes of :class:`disnake.ui.BaseSelect` (:class:`disnake.ui.Select`)
+    - subtypes of :class:`disnake.ui.BaseSelect` (:class:`disnake.ui.StringSelect`, :class:`disnake.ui.UserSelect`, :class:`disnake.ui.RoleSelect`, :class:`disnake.ui.MentionableSelect`, :class:`disnake.ui.ChannelSelect`)
```

Comment:
thoughts on ordering the selects alphabetically?

Reply:
I ordered them based on the component types (3, 5, 6, 7, 8), did the same thing for the components and items on the api reference page; I'd prefer it that way.

Reply:
I think alphabetical in all of those instances looks better and is easier to read in documentation.

Reply:
https://github.com/DisnakeDev/disnake/pull/803/commits/16e69da1647e784bdfb25b188b7fbaf1d9af4a96